api/src/main/java/org/pmiops/workbench/google/DirectoryServiceImpl.java

```diff
@@ -239,6 +252,40 @@ public void deleteUser(String username) {
     }
   }
 
+  @Override
+  public Collection<MeasurementBundle> getGaugeData() {
+    long userCount = 0;
+    try {
+      final Directory directoryService = getGoogleDirectoryService();
+      Optional<String> nextPageToken = Optional.empty();
+      do {
+        final Directory.Users.List listQuery =
+            directoryService
+                .users()
+                .list()
+                .setDomain(gSuiteDomain())
+                .setViewType(USER_VIEW_TYPE)
+                .setMaxResults(MAX_USERS_LIST_PAGE_SIZE)
+                .setOrderBy(EMAIL_USER_FIELD);
+        nextPageToken.ifPresent(listQuery::setPageToken);
+
+        final Users usersQueryResult = listQuery.execute();
+
+        userCount += usersQueryResult.getUsers().size();
```

Comment:
Do we have a place in the GSuite Admin UI where I can verify this value? Looks like 660 for fake-research-aou.org.